docs/src/pages/WebSelectPage/WebSelectPage.js

```diff
@@ -175,22 +174,11 @@ const components = [
   },
 ];
 
-const blurb = [
-  <IntroBlurb>
-    Backpack selects override the default styles in most modern browsers. In
-    some older browsers they simply fall back to the browser default.
```

Comment:
Where did this content go?

Reply:
I killed it —I was going to move it but honestly it felt redundant, the browser defaults thing is no longer a concern (we don't support any browsers where this fallback happens) and _selects override the default styles in most modern browsers._ felt like a redundant statement, that's just describing most of our components 😛 

Also, if it was that important, we don't mention it on other pages where it could be applicable (like text input)

Reply:
Makes sense 👍